app/views/site/contribute.erb

```diff
@@ -0,0 +1,130 @@
+<div class="container">
+  <section class="contributing-intro jumbotron">
+    <h1>Contributing Guide</h1>
```

Comment:
Let's not call this the Contributing Guide, since that's the official name for the files that are called `CONTRIBUTING.md` (and we have two of them... which probably is a bad idea. hm.)